modules/core/04-channel/types/msgs.go

```diff
@@ -698,16 +694,8 @@ func (msg MsgChannelUpgradeConfirm) ValidateBasic() error {
 		return errorsmod.Wrap(commitmenttypes.ErrInvalidProof, "cannot submit an empty channel proof")
 	}
 
-	if len(msg.ProofUpgradeError) == 0 {
-		return errorsmod.Wrap(commitmenttypes.ErrInvalidProof, "cannot submit an empty upgrade error proof")
-	}
-
-	if len(msg.ProofUpgradeSequence) == 0 {
-		return errorsmod.Wrap(commitmenttypes.ErrInvalidProof, "cannot submit an empty upgrade sequence proof")
-	}
-
-	if msg.CounterpartyChannel.State != OPEN {
-		return errorsmod.Wrapf(ErrInvalidChannelState, "expected: %s, got: %s", OPEN, msg.CounterpartyChannel.State)
+	if msg.CounterpartyChannelState != OPEN {
```

Comment:
I think this can be multiple values. In the linked issues I state `counterparty channel state should only be in UPGRADETRY, UPGRADEACK, or OPEN`. We can use collections.Conatins()